tests/test_minimal_requirements.txt

```
@@ -4,7 +4,6 @@ sympy==1.2
 nose==1.3.7
 nose-timer==0.7.3
 pytest~=5.2; python_version >= '3.0'
```

```suggestion
pytest~=5.2
```
Have you tried the latest pytest version (6.1.0 was just released) ?

---

I haven't. I can, though. I'll have to check out what's new and make sure stuff doesn't break

---

@neutrinoceros After a cursory investigation, it seems as if pytest 6.1.0 works